source/shared/cpp/ObjectModel/SharedAdaptiveCard.cpp

```diff
@@ -49,11 +46,13 @@ AdaptiveCard::AdaptiveCard(std::string version,
 #ifdef __ANDROID__
 std::shared_ptr<AdaptiveCard> AdaptiveCard::DeserializeFromFile(
     const std::string& jsonFile,
+    const std::string& rendererVersion,
     std::shared_ptr<ElementParserRegistration> elementParserRegistration,
     std::shared_ptr<ActionParserRegistration> actionParserRegistration) throw(AdaptiveCards::AdaptiveCardParseException)
 #else
 std::shared_ptr<AdaptiveCard> AdaptiveCard::DeserializeFromFile(
     const std::string& jsonFile,
+    double rendererVersion,
```

Comment:
this is a little bit confusing to me. who set this version?

Reply:
The renderer passes it in. This is the version of the renderer. Currently this is 1.0, and will be updated as we update our renderers.

---
In reply to: [172352845](https://github.com/Microsoft/AdaptiveCards/pull/1216#discussion_r172352845) [](ancestors = 172352845)

Reply:
thanks; that makes sense

---
In reply to: [172356287](https://github.com/Microsoft/AdaptiveCards/pull/1216#discussion_r172356287) [](ancestors = 172356287,172352845)